GLOSSARY.md

```diff
@@ -0,0 +1,128 @@
+# Glossary
+This is a glossary of terminology (possibly) used in the chalk crate.
+
+## Binary connective
+There are sixteen logical connectives on two boolean variables. The most
+interesting in this context are listed below. There is also a truth table given
+which encodes the possible results of the operations like this
+
+```
+f(false, false) f(false, true) f(true, false) f(true, true).
+```
+
+As a shorthand the resulting truth table is encoded with `true = 1` and `false =
+0`.
+
+```
+| Truth table | Operator symbol | Common name                      |
+|-------------|-----------------|----------------------------------|
+| 0001        | &&              | Conjunction; and                 |
+| 0111        | ||              | Disjunction; or                  |
+| 1001        | <=>             | Equivalence; if and only if; iff |
+| 1101        | =>              | Implication; if ... then         |
+```
```

Comment:
Since the `||` operator is not used at all in Chalk, maybe you could just drop it? I think that would avoid some confusion. Or maybe you just want to be exhaustive in a *general* way and then that's fine.